source/docs/casper/concepts/smart-contracts.md

```diff
@@ -1,7 +1,22 @@
+---
+title: Smart Contracts
+slug: /concepts/smart-contracts
+---
+
```

Comment:
This should be omitted: title is not changed and slug is the same as default URL.

```suggestion
```